docs/guides/software-lifecycle/security/security-scanning/README.md

```diff
@@ -0,0 +1,212 @@
+# Security Scanning
+
+<pre align="center">Guide to code scanning for security issues.</pre>
+
+## Introduction
+
+**Background**: Software security is critical in modern systems with application code at its root. Identifying and addressing vulnerabilities rapidly mitigates risk and limits the potential surface area of attacks. We recommend [NASA's SCRUB platform](https://github.com/nasa/scrub) to manage code scanning by identifying, orchestrating and aggregating security information. SCRUB's GitHub implementation wraps [CodeQL](https://codeql.github.com/) results into compact, curated reports that highlight security assessments and are suitable for ingestion by automated reporting tools. A small configuration is appended to an existing CodeQL configuration (`codeql-config.yml` file) that specifies security analyses and reporting properties.
+
+**Use Cases**:
```

Comment:
I think it'd be helpful to list the main driver use case for SCRUB versus just using one of the underlying tools on their own. For example "Enable interchangeability of multiple code scanning tools using a single reporting interface". Suggestion to list something like that at the top. I'm saying this because someone not knowledgable about SCRUB may wonder: _why can't I just run the code scanning tools on my own?_

Reply:
This has been addressed.